test/test-utils/tests/phase-interceptor/unit.test.ts

```diff
@@ -0,0 +1,150 @@
+import { globalScene } from "#app/global-scene";
+import type { Phase } from "#app/phase";
+import { GameManager } from "#test/test-utils/game-manager";
+import { mockPhase } from "#test/test-utils/mocks/mock-phase";
+import type { Constructor } from "#types/common";
+import type { PhaseString } from "#types/phase-types";
+import Phaser from "phaser";
+import { beforeAll, beforeEach, describe, expect, it, vi } from "vitest";
+
+// TODO: Move these to `mock-phase.ts` if/when unit tests for the phase manager are created
+class applePhase extends mockPhase {
```

Comment:
```suggestion
class ApplePhase extends mockPhase {
```
Same with the rest, classes are PascalCase.